composer.json

```diff
@@ -23,13 +23,16 @@
         }
     ],
     "require": {
-        "php": ">=5.1.2"
+        "php": "^7.1.0"
```

Comment:
This is intentional: please check http://doctrine-project.org/2017/07/25/php-7.1-requirement-and-composer.html

Reply:
@Ocramius can’t php be “^5.3.2|^7.0”?

Reply:
PHP 5 is too old but this library users seems to have existing older servers.

Reply:
Yeah, but older versions would continue to run. This release is about fixing support for stable PHP versions.

If we don't test against a PHP version, we can't consider it "supported", hence the restriction in first place.

Reply:
There is no problem if the set to --ignore-platform-reqs

Reply:
Please don't use that :-P https://twitter.com/Ocramius/status/1108306630855278592